.github/workflows/ci.yml

```
@@ -33,7 +33,7 @@ jobs:
     - name: Run tests
       run: |
         cd build
-        make coverage
+        PYTEST_OPTS="--redis-executable=redis/src/redis-server -v" make coverage BUILD_TLS=yes
```

```suggestion
        make coverage
```

We pass BUILD_TLS or PYTEST_OPTS to cmake command only. (This is how things are in cmake world, you pass parameters to cmake and it generates makefiles with these options (A bit annoying though)).